ssh-keygen: add key format conversion (#6213)
tldr-pages/tldr: pages.de/common/ssh-keygen.md

- Erstelle ein Schlüssel-Paar unter einem bestimmten Dateinamen:

`ssh-keygen -f {{~/.ssh/datei}}`

- Generiere ein ed25519 Schlüssel-Paar mit 100 Schlüssel-Ableitungs-Iterationen:

`ssh-keygen -t {{ed25519}} -a {{100}}`

- Generiere ein 4096 Bit langen RSA Schlüssel-Paar mit der Email im Kommentarfeld:

> `ssh-keygen -t {{rsa}} -b {{4096}} -C "{{email}}"`

Reviewer: To keep it in sync with the english page:
```suggestion
`ssh-keygen -t {{dsa|ecdsa|ed25519|rsa}} -b {{4096}} -C "{{kommentar|email}}""`
```

> `ssh-keygen -t {{dsa|ecdsa|ed25519|rsa}} -b {{4096}} -C "{{kommentar|email}}""`

Reviewer: Oh, sorry, I added an extra quotation mark in my suggestion
```suggestion
`ssh-keygen -t {{dsa|ecdsa|ed25519|rsa}} -b {{4096}} -C "{{kommentar|email}}"`
```

- Ändere das Passwort eines privaten Schlüssels:

`ssh-keygen -p -f ~/.ssh/{{datei}}`

> - Ändern Sie den Typ des Schlüssel formats (z. B. vom OPENSSH-Format in PEM), die Datei wird an Ort und Stelle neu geschrieben:

Reviewer: ```suggestion
- Ändere den Typ des Schlüsselformats (z. B. vom OPENSSH-Format in PEM), die Datei wird an Ort und Stelle neu geschrieben:
```
We generally don't use polite form in German pages.

Author: i used google translate for that 😄

Reviewer: Oh, in that case it's not bad 😆
